### PR TITLE
Update jackson-databind and jts

### PR DESCRIPTION
**Pull request by ahennr**
In order to get SHOGun running with actual versions of jackson-databind and geotools (which contains jts as well), this dependency needs this updates as well.

Plz review @terrestris/devs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,31 +43,30 @@ dependencies {
...
-To use JTS geometry datatypes with Jackson, you will first need to register the module first (same as
-with all Jackson datatype modules):
+To use JTS geometry datatypes with Jackson, you will first need to register the module first (same as with all Jackson
+datatype modules):
 
 ```java
-ObjectMapper mapper = new ObjectMapper();
-mapper.registerModule(new JtsModule());
+ObjectMapper mapper=new ObjectMapper();
+        mapper.registerModule(new JtsModule());
```
Comment (LukasLohoff) [inline README.md:51]: I think you don't have to change this? the old variant (`ObjectMapper mapper = new ObjectMapper();`) looks fine to me

(maybe this was an automated change?)

File: pom.xml [hunk 2]
```diff
@@ -1,186 +1,203 @@
...
-          </execution>
-        </executions>
-      </plugin>
-    </plugins>
+<!--    <scm>-->
+<!--        <connection>scm:git:git@github.com:bedatadriven/jackson-datatype-jts.git</connection>-->
+<!--        <url>scm:git:git@github.com:bedatadriven/jackson-datatype-jts.git</url>-->
+<!--        <developerConnection>scm:git:git@github.com:bedatadriven/jackson-datatype-jts.git</developerConnection>-->
+<!--        <tag>jackson-datatype-jts-2.0</tag>-->
+<!--    </scm>-->
```
Comment (LukasLohoff) [inline pom.xml:41]: can this be removed?

File: README.md (hunk 1 above)
Comment (ahennr) [reply README.md:51]: Indeed. Ill revert the automatic changes to get the diff more precise

**Comment by LukasLohoff** [comment]
Left some comments on styling. Looks good otherwise. I'm not sure about the license though, do we have to state modifications? Maybe @marcjansen can help out

**Review by LukasLohoff — APPROVED** [review]

**Review by dnlkoch — APPROVED** [review]

**Pull request merged**